### ISSUE TITLE
Validation accuracy remains below 60%

### ISSUE THREAD

**Issue by TritonSailor** [NVIDIA-Jetson/redtail#47]
Hello, I'm trying to verify I get correct results training with the forest trails dataset, using the instructions in the wiki, before I try to fine tune with my own data. However, I haven't been able to get validation accuracy above 60%.  

I tried running 40 epochs, without much difference.  Here's a few screenshots and a caffe log.  What other info can I provide to help diagnose this issue?
  

![screenshot from 2018-03-19 12-00-42](https://user-images.githubusercontent.com/4633834/37610526-d2a58d22-2b6d-11e8-93ed-efba16e87655.png)
![screenshot from 2018-03-19 12-00-04](https://user-images.githubusercontent.com/4633834/37610801-9a6776ae-2b6e-11e8-8908-516c241e8431.png)
[caffe_output(5).log](https://github.com/NVIDIA-Jetson/redtail/files/1826060/caffe_output.5.log)

**Comment by Alexey-Kamenev** [comment]
Hello,
60% validation accuracy is certainly low if you are training orientation head on original Forest Trails. With ResNet-18 you should easily get up to 85%. How did you create the dataset? I would guess you used our `idsia_trails_dataset_digits.py` with undersampling? The model that you are using looks good to me, it's the same that we used to train orientation head.
There are few options you can try:
* try creating larger dataset (oversampling)
* try using ordinary CE (cross entropy) loss instead of our custom loss by setting all parameters like `entScale` and others to 0 in `cee_loss` layer.
* try using a very small subset of your dataset to see if you can get higher accuracy.
* try changing other parameters, most notably minibatch size and learning rate.

If everything is setup correctly, here is what you should get (or something like that):

![resnet_digits](https://user-images.githubusercontent.com/8118464/37688247-4c27cc70-2c5c-11e8-8a51-614aeaeae9fa.png)

If nothing of that helps, I will try to re-train our nets again but this might not happen until April as we have GTC 2018 next week and currently a bit busy.

**Comment by TritonSailor** [comment]
Thanks for your detailed response.  Turns out the underlying issue was a corrupted Forest Trails dataset zip file.  

I downloaded the sets individually, recreated the data set with oversampling, and ran the training again.  

Results now look as expected...

![screenshot from 2018-03-23 11-17-52](https://user-images.githubusercontent.com/4633834/37840974-7b29d400-2e8c-11e8-9178-0bbb8a8acc55.png)

**Comment by gochit** [comment]
Hi @TritonSailor , I ran into the same problem. My accuracy is only 50%. I tried to rebuild the dataset using  `idsia_trails_dataset_digits.py`, but nothing change. How did you solve the problem?

BTW, I have 1 Titan X and it took me 10 hours to complete 20 epochs. Is it normal?

Thank you.
